tests/integration/artichoke.rs

```diff
@@ -0,0 +1,18 @@
+use crate::run;
+
+const BINARY: &str = "artichoke";
+const FIXTURES_ROOT: &str = "./tests/integration/apps/";
+
+#[test]
+fn test_hello_world() {
```

Comment:
usually when I write unit tests in Rust in these `test/integration` crates, I skip the `test_` prefix in the function name, like this:

https://github.com/artichoke/artichoke/blob/5f70f7ee62b903bcb62ef03f0e994771da9744f3/artichoke-backend/tests/integration/gc.rs#L16-L23